eo-maven-plugin/src/main/java/org/eolang/maven/ResolveMojo.java

```diff
@@ -202,6 +203,30 @@ private Collection<Dependency> deps() throws IOException {
             .collect(Collectors.toList());
     }
 
+    /**
+     * Check if runtime dependency is absent.
+     * @param deps Dependencies
+     * @todo #1361:90min Hardcoded version of EoRuntimeDependency.
+     *  See the EoRuntimeDependency constructor for more info.
+     *  It's much better to determine the version of the runtime library
+     *  dynamically. For example, we can fetch the latest version by http
+     *  or from config files.
+     */
+    private static void addRuntimeDependency(final Collection<Dependency> deps) {
+        if (deps.stream().noneMatch(ResolveMojo::isRuntimeDependency)) {
+            deps.add(new EoRuntimeDependency().dependency());
+        }
+    }
+
+    /**
+     * Check if dependency is runtime dependency.
+     * @param dependency Dependency
+     * @return True if dependency is runtime dependency
+     */
+    private static boolean isRuntimeDependency(final Dependency dependency) {
+        return new EoRuntimeDependency().same(dependency);
```

Comment:
@volodya-lombrozo May we create only one instance of `EoRuntimeDependency` and reuse it?

Reply:
Done